src/Request.hs

```diff
@@ -50,43 +50,43 @@ data RequestDatum = RequestDatum
     , liquidationCommission :: !Integer   -- How much % borrower will pay for lender when liquidated (before time passes)
     , requestExpiration     :: !POSIXTime
     , lenderNftTn           :: !TokenName
+    , lendDate              :: !POSIXTime
     } deriving (Show, Generic, ToJSON, FromJSON)
 
+data RequestRedeemer = RequestRedeemer {
+    lenderTn    :: !TokenName
+  , lendDateRdm :: !POSIXTime
```

Comment:
Nothing against this, just a fyi note to potentially save some tx size: It's also possible to bind it to `txValidityStart` or `txValidityEnd` (depending on the use case) instead of stating it explicitly in the redeemer

Reply:
Seconded, this is actually the cleanest and simplest solution.